src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -331,7 +331,7 @@ const computeDiff = (defaultEditSpawnHostState, editSpawnHostState) => {
     const keyToSubmit = mutationParams.publicKey;
     mutationParams.publicKey = omitTypename({
       name: keyToSubmit?.name || "",
-      key: stripNewLines(keyToSubmit.key),
+      key: stripNewLines(keyToSubmit?.key || ""),
```

Comment:
I made the caller handle the undefined case as opposed to the method since our other utils do the same.